tests/test_params.py

```diff
@@ -126,8 +126,8 @@ class ParamSpider(Args[Params], Spider):
                         "type": "boolean",
                     },
                     "fruit": {
-                        "allOf": [{"$ref": "#/$defs/FruitEnum"}],
-                        "title": "Fruit",
+                        "$ref": "#/$defs/FruitEnum",
```

Comment:
@wRAR just to make sure: does the SC UI support this format?

Reply:
Or is it the pydantic output, not the normalize schema?

Reply:
This is raw output of the latest pydantic. The normalized schema stays the same.